expertise/service/utils.py

```diff
@@ -311,37 +314,54 @@ def _camel_to_snake(camel_str):
         # Handle Group cases
         config.match_group = starting_config.get('match_group', None)
         config.alternate_match_group = starting_config.get('alternate_match_group', None)
+        config.inclusion_inv = None
+        config.exclusion_inv = None
 
         if api_request.entityA['type'] == 'Group':
             config.match_group = [api_request.entityA['memberOf']]
+            edge_inv = api_request.entityA.get('expertise', None)
+
+            if edge_inv:
+                edge_inv_id = edge_inv.get('invitation', None)
+                if edge_inv_id is None or len(edge_inv_id) <= 0:
+                    raise openreview.OpenReviewException('Bad request: Expertise invitation indicated but ID not provided')
+                if 'Inclusion' in edge_inv_id:
```

Comment:
please see my previous comment, the invitation id doesn't define if it is an inclusion or exclusion. I believe the best way to do it is inspecting the label options.